### PR TITLE
added _noop function 

### PR DESCRIPTION
**Pull request by davequick**
that allows a single instancing of all 'empty' functions for the _fileHandle lookups that are empty, then in unit tests we can simply do a equals check/===

### CODE REVIEW

File: lib/jobs/analyze-os-repo-job.js [hunk 1]
```diff
@@ -93,6 +93,15 @@ function analyzeOsRepoJobFactory(
     };
 
     /**
+     * A function that can be used to provide a function with no poerations or side effects when lookup fails.
```
Comment (davequick) [inline lib/jobs/analyze-os-repo-job.js]: cool - i already like this hound thing.  handy.

**Comment by benbp** [comment]
Squash please?

File: lib/jobs/analyze-os-repo-job.js [hunk 2]
```diff
@@ -93,6 +93,16 @@ function analyzeOsRepoJobFactory(
     };
 
     /**
+     * A function that can be used to provide a function with no poerations or side effects
```
Comment (benbp) [inline lib/jobs/analyze-os-repo-job.js]: "poerations"

**Comment by benbp** [comment]
FYI @yyscamper

**Comment by benbp** [comment]
:+1:

**Comment by DavidjohnBlodgett** [comment]
:+1:

**Comment by heckj** [comment]
+1

**Comment by yyscamper** [comment]
Sorry to miss this PR review. Code is good, :+1:

**Pull request merged**